lib/linguist/languages.yml

```diff
@@ -1703,6 +1703,16 @@ Limbo:
   tm_scope: none
   ace_mode: text
 
+Linker Script:
+  type: data
```

Comment:
Because, even though it's called a script, the language doesn't provide for programming constructs.  I.e. no conditionals, variables, loops, procedures, etc.